tests/NormalDistributionTest.php

```diff
@@ -71,6 +71,15 @@ public function testNormalPDF()
         $this->assertEquals($d->pdf(-0.5), 0.3520653267, "PDF incorrect", 1e-9);
         $this->assertEquals($d->pdf(1.5), 0.1295175956, "PDF incorrect", 1e-9);
         $this->assertEquals($d->pdf(3.5), 0.0008726826947, "PDF incorrect", 1e-9);
+
+        $d = new Normal(0, 0.2);
+
+        $this->assertEquals(6.4119473711506E-28, $d->pdf(-5), "PDF incorrect", 1e-9);
+        $this->assertEquals(1.460642012962E-7, $d->pdf(-2.5), "PDF incorrect", 1e-9);
+        $this->assertEquals(0.89206205807639, $d->pdf(0), "PDF incorrect", 1e-9);
+        $this->assertEquals(0.47748641153356, $d->pdf(-0.5), "PDF incorrect", 1e-9);
```

Comment:
1) Mathematica from Wolrfam 

PDF[NormalDistribution[0, Sqrt[0.2]], -0.5] -> 0.477486

2) https://www.danielsoper.com/statcalc/calculator.aspx?id=54

for Parameters: mean 0, standard derivation sqrt[0.2] = 0.447214 and x = -0.5 gives PDF = 0.47748545

In this test 0.47748641153356 so it is ok too.